content/v2.0/get-started.md

```diff
@@ -8,13 +8,161 @@ weight: 1
+When using Telegraf, the InfluxDB v2 API, the `influx` CLI, or the client libraries
+to interact with your {{< cloud-name "short" >}}, they each require the URL of your
+InfluxDB Cloud instance.
+See [InfluxDB Cloud URLs](/v2.0/cloud/urls/) for information about which URL to use
+and where to find it in your InfluxDB Cloud UI.
+{{% /note %}}
+
+## Query and visualize data
+
+Once you've set up {{< cloud-name "short" >}} to collect data, you can do the following:
+
+- Query data using Flux, the UI, and the `influx` command line interface.
+  See [Query data](/v2.0/query-data/).
+- Build custom dashboards to visualize your data.
+  See [Visualize data](/v2.0/visualize-data/).
+
+## Process data
+
+Use InfluxDB tasks to process and downsample data. See [Process data](/v2.0/process-data/).
+
+## View data usage
+
+Once you're up and running with {{< cloud-name "short" >}}, [monitor your data usage in
+your {{< cloud-name "short" >}} UI](/v2.0/cloud/account-management/data-usage/).
+
+## Differences between InfluxDB Cloud and InfluxDB OSS
+
+{{< cloud-name >}} is API-compatible and functionally compatible with InfluxDB OSS 2.0.
+The primary differences between InfluxDB OSS 2.0 and InfluxDB Cloud 2.0 are:
+
+- [InfluxDB scrapers](/v2.0/write-data/scrape-data/) that collect data from specified
+  targets are not available in {{< cloud-name "short" >}}.
+- {{< cloud-name "short" >}} instances are currently limited to a single organization with a single user.
+- Retrieving data from a file based CSV source using the `file` parameter of the
+  [`csv.from()`](/v2.0/reference/flux/functions/csv/from) function is not supported;
+  however you can use raw CSV data with the `csv` parameter.
+
+### New features in InfluxDB Cloud 2.0
+
+- **Free Plan (rate-limited)**: Skip downloading and installing InfluxDB 2.0 and
+  jump right in to exploring InfluxDB 2.0 technology.
+  The Free Plan is designed for getting started with InfluxDB and for small hobby projects.
+- **Flux support**: [Flux](/v2.0/query-data/get-started/) is a standalone data
+  scripting and query language that increases productivity and code reuse.
+  It is the primary language for working with data within InfluxDB 2.0.
+  Flux can be used with other data sources as well.
+  This allows users to work with data where it resides.
+- **Unified API**: Everything in InfluxDB (ingest, query, storage, and visualization)
+  is now accessible using a unified [InfluxDB v2 API](/v2.0/reference/api/) that
+  enables seamless movement between open source and cloud.
+- **Integrated visualization and dashboards**: Based on the pioneering Chronograf project,
+  the new user interface (InfluxDB UI) offers quick and effortless onboarding,
+  richer user experiences, and significantly quicker results.
+- **Usage-based pricing**: The [The Usage-based Plan](/v2.0/cloud/pricing-plans/#usage-based-plan)
+  offers more flexibility and ensures that you only pay for what you use.
+
+# Start using InfluxDB OSS
```

Comment:
## Get started with InfluxDB OSS 2.0

or update Cloud header to match....## Start using InfluxDB Cloud 2.0